### ISSUE TITLE
Cannot coerce type 'KeyCodeConstants' to 'Long'

### ISSUE THREAD

**Issue by Kr00l** [WaynePhillipsEA/twinbasic#185]
**Describe the bug**
When passing a 'KeyCodeConstants' variable to an API which expects a 'Long' the error is "Cannot coerce" is produced.

**To Reproduce**
```
Private Declare Function GetKeyState Lib "user32" (ByVal nVirtKey As Long) As Integer

Public Function KeyToggled(ByVal KeyCode As KeyCodeConstants) As Boolean
KeyToggled = CBool(LoByte(GetKeyState(KeyCode)) = 1)
End Function
```

**Expected behavior**
That the variable can be coerced to 'Long', like in VBx.

**Screenshots**
![image](https://user-images.githubusercontent.com/33756666/120767758-c7b13380-c51b-11eb-9773-1ba986c90646.png)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
 - twinBASIC compiler version [e.g. 0.10.933]

**Additional context**
n/a

**Comment by WaynePhillipsEA** [comment]
The reason for this is because in VBA, KeyCodeConstants is a module, whereas in VB6 it is an enum.   twinBASIC is currently treating it as a module.  I'll switch it to an enum.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.1066, thanks for reporting.
